docs/terms/store.md

```diff
@@ -3,19 +3,107 @@ title: Store
 id: store
 hoverText: A connector to store and retrieve information about metadata in Great Expectations.
 ---
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import SetupHeader from '/docs/images/universal_map/_um_setup_header.mdx'
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='active' connect='inactive' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
 
 A Store is a connector to store and retrieve information about metadata in Great Expectations.
 
-Great Expectations supports a variety of Stores for different purposes, but the most common Stores are Expectation Stores, Validations Stores, Checkpoint Stores, and Evaluation Parameter Stores (or Metric Stores). 
+### Features and promises
+
+Great Expectations supports a variety of Stores for different purposes, but the most common Stores are Expectation Stores, Validations Stores, Checkpoint Stores, and Evaluation Parameter Stores (or Metric Stores).  Data Docs Stores can also be configured for Data Doc Sites.  Each of these Stores is tailored to a specific type of information.
```

Comment:
Could you explain "Evaluation Parameter Stores (or Metric Stores)" in a bit more detail?  It looks like the page on Metric Stores distinguishes the two, so it seems contradictory to equate them here, and differentiate them there.

Reply:
That is in fact a typo.  I had received some documentation when I was consolidating sources that implied the two were functionally the same.  Later I received some clarification that they are not due to how each stores the information they contain; I updated the doc but must have overlooked this spot that had them conflated still.  I've split them apart now.